CODEOWNERS

```diff
@@ -31,6 +31,26 @@
 /connectivity/check/ipcache.go @cilium/ipcache
 /connectivity/check/metrics*.go @cilium/metrics
 /connectivity/check/policy.go @cilium/sig-policy
+/connectivity/builder/** @cilium/ci-structure
+/connectivity/builder/all_ingress_deny_from_outside.go @cilium/sig-encryption
+/connectivity/builder/cluster_entity_multi_cluster.go @cilium/sig-clustermesh
+/connectivity/builder/dns_only.go @cilium/sig-clustermesh
+/connectivity/builder/echo_ingress.go @cilium/sig-clustermesh
+/connectivity/builder/echo_ingress_auth_always_fail.go @cilium/sig-clustermesh
+/connectivity/builder/echo_ingress_from_other_client_deny.go @cilium/sig-clustermesh
+/connectivity/builder/echo_ingress_from_outside.go @cilium/sig-clustermesh
+/connectivity/builder/echo_ingress_knp.go @cilium/sig-clustermesh
+/connectivity/builder/echo_ingress_l7.go @cilium/sig-clustermesh
+/connectivity/builder/echo_ingress_l7_named_port.go @cilium/sig-clustermesh
+/connectivity/builder/echo_ingress_mutual_auth_spiffe.go @cilium/sig-encryption
```

Comment:
Based on the features these are testing, I think they should be assigned to @cilium/sig-servicemesh instead.

Reply:
Fixed.